src/components/footer/_footer.scss

```diff
@@ -30,13 +30,13 @@
         vertical-align: middle;
     }
 
-    &__poweredBy-link {
-        &:focus {
-            .ons-icon--logo {
-                @extend %a-focus;
-            }
-        }
+  &__poweredBy-link {
+    &:focus {
+      .ons-icon--logo {
+        background-color: var(--ons-color-focus);
+      }
     }
+  }
```

Comment:
Looks like this change is removing the underline to the default logo. 
![Screenshot 2024-06-04 at 14 39 31](https://github.com/ONSdigital/design-system/assets/112873190/3d9d6258-d67a-460b-a373-c619e229e584)